README.md

```diff
@@ -14,12 +14,36 @@ An integrated application hosting platform which allows you to host your SPAs. I
 - Notifications Framework
 - Feedback Framework
 
+#### Install prerequisites
+
+- Docker
+
 ## Install
 
+### Install node_modules for all the packages
 ```sh
 npm install
-# OR
-lerna bootstrap
+npm run lerna bootstrap
```

Comment:
Can you add the lerna bootstrap to package.json > `scripts.postinstall`?\
That will reduce the number of steps.

Reply:
This would be needed in one of the other commands and for manual installation @deshmukhmayur